### PR TITLE
Docker "Best Practices"

### PR DESCRIPTION
**Pull request by Joly0**
I guess this should cover most of the best practices. This could be further improved overall, for example to a Run one-liner, but thats ok for now. "apt-get -y autoremove --purge" might be a bit overkill, but we´ll see.

### CODE REVIEW

**Comment by MitchTalmadge** [comment]
Thanks! I will look this over more once I get time to test it. I have a busy week :P

**Comment by Joly0** [comment]
Alright. Yeah, the mutli stage builds are usefull to develop, deploy and minimize the image, but the advantage is only archieved, when you copy everything you build to a new scratch image at the end.

File: Dockerfile [hunk 1]
```diff
@@ -8,70 +8,113 @@ ENV PASSWORD=password
 ENV LICENCE=notset
 ENV MODULE=ADS
 
+ARG DEBIAN_FRONTEND noninteractive
```
Comment (MitchTalmadge) [inline Dockerfile:11]: This line is missing an `=` (`ARG DEBIAN_FRONTEND=noninteractive`) but I will fix that :)

**Review by MitchTalmadge — APPROVED** [review]

**Comment by MitchTalmadge** [comment]
Thanks it's great!

**Pull request merged**